doc/source/getting_started.rst

```diff
@@ -0,0 +1,58 @@
+Getting started
+###############
+
+Installation
+============
+
+This package may be installed following two procedures: either the pip package
+manager installation or the manual installation. The process to be followed for
+each of them is shown in the upcoming sections.
+
+The ``pyansys-tools-versioning`` package currently supports Python >=3.7 on
+Windows, Mac OS, and Linux.
+
+Install the latest release from PyPi with:
+
+.. code-block:: bash
+
+   python -m pip install pyansys-tools-versioning
+
+Alternatively, install the latest from GitHub via:
+
+.. code-block:: bash
+
+   python -m pip install git+https://github.com/pyansys/pyansys-tools-versioning.git
+
+For a local development version, install with (requires pip >= 22.0):
+
+.. code-block:: bash
+
+   git clone https://github.com/pyansys/pyansys-tools-versioning.git
+   cd pyansys-tools-versioning
+   python -m pip install -e .
+
```

Comment:
I think that when dealing with ``pyproject.toml`` projects, you cannot install it in editable mode, right?

Reply:
There used to be a problem with this some years ago. However, `pip` latest releases support editable installs for packages using a `pyproject.toml`.

Reply:
I guess I need to update my ``pip`` 😄